### DISCUSSION TITLE
【table】可编辑单元格怎么在父组件中调用components.body.cell组件的方法或数据

### DISCUSSION THREAD

**Discussion by peacejj** [ant-design/ant-design#34113] [General]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
现在表格外面不能调用验证方法。

### What does the proposed API look like?
期望可以在表格外面调用验证方法

**Comment by zjj1019** [comment]
用 EditableRow 的form可以全局校验，传个ref过来就行
